### DISCUSSION TITLE
packagecloud reporting 404 for all ubuntu packagecloud repos

### DISCUSSION THREAD

**Discussion by lrosenman** [rabbitmq/rabbitmq-server#8384] [Other] [answered]
### Describe the bug

Did packagecloud delete/turn off the rabbitmq repos?

### Reproduction steps

follow install guide, and get apt update failure

### Expected behavior

apt update retrieve packages

### Additional context

this started today (27-May-2023 at 13:00 CDT (UTC-5))

**Comment by michaelklishin** [comment, marked as answer]
There were card on file charge failures. Use a [Cloudsmith mirror](https://rabbitmq.com/install-debian.html#apt-quick-start-cloudsmith).

**Comment by michaelklishin** [comment]
I'm being told that due to the nature of the problem (corporate bureaucracy) and the long weekend in the U.S. this can take more than a week.

Simply switch to the Cloudsmith mirror, it has no traffic quotas and is not subject to a myriad of bureaucratic procedures.

I'm sorry about this. This stuff is incredibly frustrating to the core team.
